### PR TITLE
add governance docs

### PR DESCRIPTION
**Pull request by FredKSchott**
## Changes

- Adds code of conduct
- Adds governance doc
- Adds community doc (still WIP but not blocking this PR, should eventually become a hosted page on our docs site)

Note: Any existing contributors with write access (granted post-launch) will be grandfathered in to become L2 contributors. All other contributors will become L1.

/cc @snowpackjs/maintainers for review

### CODE REVIEW

**Comment by jasikpark** [comment]
I don't have much input, but looks great to me!

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+Outlined below is the process for Code of Conduct violation reviews.
+
+### Reporting
+
+Anyone may report a violation. Violations can be reported in the following ways:
+
+- In private, via email to one or more stewards.
+- In private, via direct message to a project steward on Discord.
+- In public, via a GitHub comment (mentioning `@rometools/stewards`).
```
Comment (ewatch) [inline GOVERNANCE.md:136]: I think this needs to be changed to @snowpackjs/astro/stewards or something like that.

**Comment by ewatch** [comment]
Looks great to me as well.
It's understandable and relative concrete.

Some words leave space for interpretation through different people (i.e. non-trivial).

**Comment by Hanawa02** [comment]
Looks good to me.

File: GOVERNANCE.md (hunk 1 above)
Comment (FredKSchott) [reply GOVERNANCE.md:136]: lol shoot thought I got them all

**Comment by natemoo-re** [comment]
Looks great! The only big question I have is about roles. It would make sense to have roles (teams) like `@snowpackjs/stewards`, `@snowpackjs/core`, `@snowpackjs/contributors-L3`, `L2`, etc *but* this would mean that Astro/Snowpack contributors are one and the same... any ideas?

**Comment by jasikpark** [comment]
Looks like a job for Teams?     https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/giving-team-maintainer-permissions-to-an-organization-member
  
​
  
​Just learned about it now xD
  
​
  
  
  
Caleb Jasik
  
***@***.***
  
​https://jasik.xyz

**Comment by jasikpark** [comment]
I felt dumb because I imagine that's exactly what you're probably talking about... but it seems that you can nest teams, maybe that works?     https://docs.github.com/en/organizations/organizing-members-into-teams/requesting-to-add-a-child-team
  
​
  
  
  
Caleb Jasik
  
***@***.***
  
​https://jasik.xyz

**Comment by FredKSchott** [comment]
I haven't talked through this yet with Drew, but I'd actually like to merge Snowpack and Astro governance under one roof. Becoming a contributor in one would make you a contributor in all.

Note that this wouldn't mean everyone gets to push code wherever they want :) All committers still need to go through the review process to get anything merged to `main`.

**Comment by jasikpark** [comment]
I feel like it'd still be useful to define subteams for declaring certain people as part of a repo?

**Comment by FredKSchott** [comment]
We've had `@snowpackjs/docs` in the past, could be worth revisiting.

This isn't really captured in the governance docs, but I like the idea of having a contributor **level** & **focus**. 
- Level is === how much experience you have in the project
- Focus is === where your expertise lies. not everyone is an expert in everything!

So the idea is that you could be a member of @snowpack/contributor-core & @snowpack/docs.

**Comment by FredKSchott** [comment]
Merging now to get something out there, see note on top of doc about letting this bake a bit longer before it becomes binding. If you run into any issues, lmk! Also, if anything feels too high friction (or even worse, that we don't follow some process even though we said we would) also call that out. It could be early growing pains, or it could be that the process needs to change in some way to be easier to follow.

I'll go ahead and create some new GH teams & Discord roles tomorrow!

**Pull request merged**